src/backup.ts

```diff
@@ -56,6 +57,19 @@ export interface CreateBackupOptions {
   gaxOptions?: CallOptions;
 }
 
+export interface CopyBackupGaxOperation extends GaxOperation {
+  // Overridden with more specific type for CreateBackup operation
+  metadata: Metadata &
+    databaseAdmin.spanner.admin.database.v1.ICopyBackupMetadata;
+}
+
+export type CopyBackupResponse = [Backup, CopyBackupGaxOperation, IOperation];
+
+export interface CopyBackupOptions
+  extends databaseAdmin.spanner.admin.database.v1.ICopyBackupRequest {
+  gaxOptions?: CallOptions;
+}
```

Comment:
@olavloite what are your thoughts on our design choice? For BackupOptions, we created a new object (not extending IBackupRequest). But this ended up being a poor choice because every time a new field was added, we'd have to modify the interface. So I asked @asthamohta to do it this way. The trouble now is that create and copy have different inputs. E.g. expireTime needs to be specified as a proto in the latter.

Reply:
I agree that extending `IBackupRequest` is probably the better choice here. At least our experiences with other features where we used the proto interfaces have been good.

Would it maybe be possible to change the `createBackup` function so it accepts both the current `CreateBackupOptions` and a new one that extends `ICreateBackupRequest`? (And deprecate the current `CreateBackupOptions` definition.) That way this design choice might feel less as a deviation than now.

Reply:
I think we can get it to accept an `ICreateBackupRequest` but it may not work very well because that proto doesn't have a field for gaxOptions. So we'll have to introduce a new interface. Then it might be confusing for users.

As for deprecating `CreateBackupOptions`, I don't think we can do that because that would be a breaking change :(